client/app/dedicatedCloud/dedicatedCloud.routes.js

```diff
@@ -2,6 +2,8 @@ angular
   .module('App')
   .config(/* @ngInject */ ($stateProvider, $urlServiceProvider) => {
     $stateProvider.state('app.dedicatedClouds', {
+      reloadOnSearch: false,
```

Comment:
Imo it's easier to have the `url` first in terms of reading even though alphabetical order seems logical as you don't always have the same properties (you can have `controller`, `views`, `layout` ...)